.github/workflows/ci.yml

```
@@ -91,6 +91,11 @@ jobs:
         with:
           go-version-file: go.mod
           cache: true
+      - name: Install ginkgo
+        run: |
+          command -v ginkgo || go install github.com/onsi/ginkgo/ginkgo@${{ env.GINKGO_VERSION }}
+        env:
+          GINKGO_VERSION: "v2.8.0"
```

```suggestion
          GINKGO_VERSION: v2.8.0
```

---

If you want this to get automatic dependency update suggestions:

```suggestion
          # renovate: datasource=github-tags depName=onsi/ginkgo
          GINKGO_VERSION: v2.8.0
```

Not possible to use `# renovate: datasource=go depName=github.com/onsi/ginkgo/ginkgo` since we have disabled golang for Renovate.

---

its a temporary change to see if ginkgo provides anything useful to debug the flakiness. So lets skip renovator for now :)